action.yml

```diff
@@ -17,6 +17,11 @@ inputs:
     required: false
     default: ""
 
+  use-cache:
```

Comment:
Can we call it `use-cache-if`?

Reply:
We can call it supercalifragilisticexpialidocious if you want, it's up to you. :D